evaluations/run_evaluation.py

```diff
@@ -28,16 +28,24 @@
 
 from wfa_cardinality_estimation_evaluation_framework.evaluations import analyzer
 from wfa_cardinality_estimation_evaluation_framework.evaluations import evaluator
-from data import evaluation_configs
+from wfa_cardinality_estimation_evaluation_framework.evaluations import report_generator
+from wfa_cardinality_estimation_evaluation_framework.evaluations.data import evaluation_configs
```

Comment:
This should be:
from data import evaluation_configs
because data is not part of the evaluations package anymore.